src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -57,6 +57,10 @@ bool SupportMKLDNNAct(const ActivationParam& param, const NDArray &input) {
   return SupportMKLDNNAct(param);
 }
 
+bool SupportQuantizedMKLDNNAct(const ActivationParam &param) {
+  return param.act_type == activation::kReLU;
```

Comment:
Add `TODO` for other FP32 fusion in the future

Reply:
Done.